.circleci/config.yml

```diff
@@ -647,6 +650,7 @@ workflows:
             - ui-unit-test
       # On master branch merges, run Puppeteer tests after ui and api deployed to "test" env successfully.
       - puppeteer-e2e-test:
+          parallel_num: 4
```

Comment:
Up parallelism from 3 to 4 in order to reduce e2e test duration time ~15 minutes.